build.gradle

```diff
@@ -31,6 +31,7 @@ apply plugin: "io.sdkman.vendors"
 apply plugin: 'org.aim42.htmlSanityCheck'
 
 repositories {
+    mavenLocal()
```

Comment:
Please remove mavenLocal()

Reply:
Yeah, sorry, left it from the local testing I was doing. Fixed now.